src/gui/guiutility.cpp

```diff
@@ -22,13 +22,22 @@
 #include <QUrlQuery>
 
 #include "common/asserts.h"
-
 using namespace OCC;
 
 Q_LOGGING_CATEGORY(lcUtility, "nextcloud.gui.utility", QtInfoMsg)
 
 bool Utility::openBrowser(const QUrl &url, QWidget *errorWidgetParent)
 {
+    const QStringList whitelistedUrlSchemes = {
```

Comment:
nitpick: rename to `allowedUrlSchemes`

Reply:
@er-vin Ok. Will fix CI errors later, before merge...